scripts/measure-jitter.js

```
@@ -0,0 +1,164 @@
+  cyan: '\x1b[36m'
+};
+const colorize = (txt, clr) => `${COLORS[clr] ?? ''}${txt}${COLORS.reset}`;
+
+// Configuration
+const ENDPOINT = 'https://cb.apigw.io/ping';
+const REQUESTS = 20; // Number of requests to make
+const INTERVAL = 200; // ms between requests
+
+// Statistics
+const latencies = [];
+let completed = 0;
```

ESLint

[error] 'completed' is assigned a value but never used.

(no-unused-vars)